projects/gfbio-app/src/app/app.component.html

```diff
@@ -0,0 +1,53 @@
+<mat-sidenav-container fullscreen>
+    <!--    <mat-sidenav position="end" mode="side" (openedChange)="mapComponent.resize()">-->
+    <mat-sidenav position="end" mode="side">
+        <wave-sidenav-container></wave-sidenav-container>
+    </mat-sidenav>
+    <div class="time-container" fxLayoutAlign="space-around center">
+        <wave-zoom-handles (zoomIn)="map.zoomIn()" (zoomOut)="map.zoomOut()" class="mat-elevation-z4"></wave-zoom-handles>
+        <wave-small-time-interaction class="mat-elevation-z4"></wave-small-time-interaction>
+        <!--        <wave-ticker-interaction *ngIf="config.COMPONENTS.PLAYBACK.AVAILABLE" class="mat-elevation-z4"></wave-ticker-interaction>-->
+        <div></div>
```

Comment:
![grafik](https://user-images.githubusercontent.com/4947061/130461538-c74f12ba-fc1b-4376-922b-2100c169b021.png)

I guess the splash screen it not 100% up-to-date and there is no back button for GFBio.

Reply:
the green box doesn't make sense because we have not yet integrated the search results yet. I added the back button 0b98e8eb72d8e07b75d819f61550225bbc29ad7d